compiler/src/dotty/tools/dotc/core/tasty/TastyPrinter.scala

```diff
@@ -47,7 +49,8 @@ class TastyPrinter(bytes: Array[Byte])(implicit ctx: Context) {
     def unpickle(reader: TastyReader, tastyName: TastyName.Table): Unit = {
       import reader._
       var indent = 0
-      def newLine() = print(f"\n ${index(currentAddr) - index(startAddr)}%5d:" + " " * indent)
+      val length = "%5d:".format(index(currentAddr) - index(startAddr))
```

Comment:
Since `newLine` is called twice and `length` is defined as a val, then I believe length would have the same value even though `currentAddr` and `startAddr` might mutate. This should therefore be inside the def like:

```scala
def newLine() = {
  val length = "%5d:".format(index(currentAddr) - index(startAddr))
  print(s"\n $length" + " " * indent)
}
```